javascript-create-module/template/README.md

```diff
@@ -10,22 +10,30 @@ If you don't use default configuration for the Docker container name or for Jahi
 
 You can find the documentation on how to use this module on the [Jahia Academy](https://academy.jahia.com/get-started/developers/templating) templating tutorial.
 
-## Run
+## Prerequisites
 
-1. Enable Corepack if needed :
+Yarn is required to build the project.
 
-`enable corepack`
+Please follow the instructions at [Node.js Downloads](https://nodejs.org/en/download) to install Node.js and Yarn. In the dropdown menus, make sure to select Yarn (the _with_) and your operating system (the _for_).
 
-2. Install the dependencies :
+Note: If you are using a Windows machine, you also need to install PowerShell to run the scripts in `package.json` (which use `mkdir`, `rm`, etc.). You can download it from [PowerShell GitHub](https://github.com/PowerShell/PowerShell).
```

Comment:
```suggestion
Note: If you are using a Windows machine, you need to run your `yarn` commands in PowerShell (not cmd.exe!). It comes preinstalled on modern Windows versions. In case this is not the case on your machine, [please see Microsoft documentation](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-windows?view=powershell-7.5).
```

Reply:
oops, windows newbie error :)